packages/wrangler/src/dev/miniflare.ts

```diff
@@ -442,6 +442,9 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 			serviceBindings[service.binding] = {
 				name: service.service,
 				entrypoint: service.entrypoint,
+				props: service.props
+					? { json: JSON.stringify(service.props) }
+					: undefined,
```

Comment:
I don't think this is the right place to do this. IMO props should be passed as an object to Miniflare and then stringified before they're sent to workerd